system-validation-test/features/step_definitions/rest_steps.rb

```diff
@@ -119,6 +119,10 @@ def get_url(concept_type, cmr_root)
   @body = xml
 end
 
+Given(/^I clear the body$/) do
+  @body = ''
```

Comment:
Would prefer if this was set to `nil` instead of an empty string since that could still be considered a payload

Reply:
Fixed